wp-content/themes/bzLLKit/index.php

```diff
@@ -33,10 +33,10 @@
 			// to view courses they're associated with on the Portal,
 			// by adding an argument to the query:
 
-			$current_user_email = wp_get_current_user()->user_email;
+			// $current_user_email = wp_get_current_user()->user_email;
 			// FOR TESTING:
-			$current_user_email = 'aalcones@fb.com';
-
+			// $current_user_email = 'aalcones@fb.com';
```

Comment:
If this is a real person's email. Can you delete it? We shouldn't have it checked into public src ctrl.